### DISCUSSION TITLE
Options Override for WYSIWYG works for width but not height?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#11813] [Archive]
In order to get a smaller initial height for a WYSIWYG, according to [TinyMCE documentation](https://www.tiny.cloud/docs/general-configuration-guide/customize-ui/#changingeditorheightandwidth), I set
**Options Override**
`{ "height": "200" }`

But it has no effect.

On the contrary, setting the width is effective
**Options Override**
`{ "width": "300" }`

What would be the right option to override the height?

**Comment by azrikahar** [comment]
The option should be correct, but there's a `.tox-tinymce` style setting `min-height` to `300px`. You should be able to override that via Project Settings -> Custom CSS and set it to `auto !important`.

![Pyf3VOefk1](https://user-images.githubusercontent.com/42867097/155280597-a28c3585-4f4f-44c0-b9de-eac746f0db09.gif)

**Reply by Pachat** [reply]
Thanks for the quick reply and the complete explanation.
It acts on all WYSIWYG. That's OK. 

I ended with nothing in Options Override as it would not change the height for a specific WYSIWYG (albeit working for width):

- The problem is whatever `height` is in Options Override, it is overridden somewhere by `200px` 
when `.tox-tinymce` style is set to `auto !important` in the Project Settings
